common/version_test.go

```diff
@@ -49,39 +49,39 @@ func (vSuite *versionTestSuite) TestVersionEquality() {
 
 	v1, _ := ParseVersion("10.0.0")
 	v2, _ := ParseVersion("10.0.0")
-	assert.Equal(v1.compare(*v2), 0)
+	assert.Equal(0, v1.compare(*v2))
```

Comment:
Did the linter make these changes as well? Seems like a trivial change here.

Reply:
Yes. These changes to Equal are the most important ones in the PR. The Equal function expects that the first values is the expected value and the second is the actual value. When the tests are passing it doesn't matter the order, but when I test fails then the order being wrong will mislead you. Because the error message you will get is backward and it will tell you the wrong expected value and the wrong actual value.